action.yml

```diff
@@ -77,6 +85,7 @@ runs:
         GIT_COMMITTER_NAME: ${{ steps.config.outputs.git-committer-username }}
         GIT_COMMITTER_EMAIL: ${{ steps.config.outputs.git-committer-email }}
         CURRENT_VERSION: ${{ steps.config.outputs.current-version }}
+        VERSION_FILEPATH: ${{ steps.config.outputs.version-filepath }}
```

Comment:
`RELATIVE_GRADLE_FILEPATH` would be better

Reply:
Agreed, assuming the change is not done on the Maven part of the tool. If it is, then the new name is fine.